README.md

```diff
@@ -8,9 +8,9 @@
 [![Total Downloads](https://poser.pugx.org/box/spout/downloads)](https://packagist.org/packages/box/spout)
 
 Spout is a PHP library to read and write spreadsheet files (CSV, XLSX and ODS), in a fast and scalable way.
-Contrary to other file readers or writers, it is capable of processing very large files while keeping the memory usage really low (less than 3MB).
+Unlike other file readers or writers, it is capable of processing very large files, while keeping, the memory usage really low (less than 3MB).
```

Comment:
does it need 2 commas here? the one after "keeping" looks weird to me.

Reply:
No sorry it doesnt , I was reading it earlier an thought it improves it but doesnt, I'll update.